### PR TITLE
[GPU] Performance counters fix

### PR DESCRIPTION
**Pull request by Lyamin-Roman**
### Tickets:
 - *51819*

### CODE REVIEW

**Review by tsavina — APPROVED** [review]

File: inference-engine/thirdparty/clDNN/src/program.cpp [hunk 1]
```diff
@@ -937,7 +937,7 @@ bool program::extract_and_remove(program_node& node) {
         outputs.erase(std::remove(outputs.begin(), outputs.end(), &node), outputs.end());
 
         rename(node, "_cldnn_tmp_" + node_id);
-        rename(prev, node_id);
+        add_optimized_primitive_info(prev.id(), { node_id });
```
Comment (sshlyapn) [inline inference-engine/thirdparty/clDNN/src/program.cpp:940]: Is it correct usage of function? As I can see, the first argument is `primitive_id optimized_primitive_id` and the second one is `std::vector<primitive_id> replaced_with_ids`, so you should swap the arguments

File: inference-engine/thirdparty/clDNN/tests/test_cases/convolution_gpu_test.cpp [hunk 2]
```diff
@@ -5520,7 +5527,7 @@ TEST(convolution_f32_fw_gpu, convolution_int8_b_fs_yx_fsv4_to_bfyx) {
         reorder("to_int", "input", { data_types::i8,format::b_fs_yx_fsv4,{ batch_num, input_f, input_size_x, input_size_y } }),
         data("weights", weights),
         data("biases", biases),
-        convolution("conv", "to_int", { "weights" }, { "biases" }, { 1, 1, 1, 1 }, { 0, 0, -2, -2 }, { 1,1,1,1 },
+        convolution("conv", "to_int", { "weights" }, { "biases" }, { 1, 1, 1, 1 }, { 0, 0, -2, -2 }, { 1,1,1,1 }, "",
```
Comment (lznamens) [inline inference-engine/thirdparty/clDNN/tests/test_cases/convolution_gpu_test.cpp:5530]: Spaces between numbers.

File: inference-engine/thirdparty/clDNN/tests/test_cases/convolution_gpu_test.cpp [hunk 3]
```diff
@@ -5498,7 +5505,7 @@ TEST(convolution_f32_fw_gpu, convolution_int8_b_fs_yx_fsv4_to_bfyx) {
         reorder("to_int", "input", { data_types::i8,format::bfyx,{ batch_num, input_f, input_size_x, input_size_y } }),
         data("weights", weights),
         data("biases", biases),
-        convolution("conv", "to_int", { "weights" }, { "biases" }, { 1, 1, 1, 1 }, { 0, 0, -2, -2 }, { 1,1,1,1 },
+        convolution("conv", "to_int", { "weights" }, { "biases" }, { 1, 1, 1, 1 }, { 0, 0, -2, -2 }, { 1,1,1,1 }, "",
```
Comment (lznamens) [inline inference-engine/thirdparty/clDNN/tests/test_cases/convolution_gpu_test.cpp:5508]: Spaces between numbers.

File: inference-engine/thirdparty/clDNN/tests/test_cases/activation_simple_gpu_test.cpp [hunk 4]
```diff
@@ -984,7 +984,7 @@ TEST(activation_f32_fw_gpu, relu_basic_acosh_yxfb) {
     topology topology(
             input_layout("input", input->get_layout()),
             reorder("reorder", "input", input->get_layout().with_padding(padding{{0, 0, 2, 1}, 0})),
-            activation("relu", "reorder", activation_func::acosh, {0.5f, 0.f}, padding{{0, 0, 0, 0}, 0}));
+            activation("relu", "reorder", activation_func::acosh, {0.5f, 0.f}, "", padding{{0, 0, 0, 0}, 0}));
```
Comment (lznamens) [inline inference-engine/thirdparty/clDNN/tests/test_cases/activation_simple_gpu_test.cpp:987]: Please use unified code style as in other tests in this file and add spaces inside figure braces.

File: inference-engine/src/cldnn_engine/cldnn_graph.cpp [hunk 5]
```diff
@@ -599,13 +569,21 @@ std::map<std::string, InferenceEngine::InferenceEngineProfileInfo> CLDNNGraph::G
...
-    auto getUpperCaseName = [&](std::string name) {
+    auto getUpperCaseName = [](std::string name) {
         if (name.length() > 0)
             name[0] = toupper(name[0]);
         return name;
     };
 
+    auto getClearName = [](std::string name) {
+        if (name.find(":") != std::string::npos) {
+            name = name.substr(name.find(":") + 1, name.length());
```
Comment (lznamens) [inline inference-engine/src/cldnn_engine/cldnn_graph.cpp:582]: Does this simple solution really cover all cases?

**Review by lznamens — CHANGES_REQUESTED** [review]

**Review by lznamens — APPROVED** [review]

**Pull request merged**